config/test.exs

```diff
@@ -0,0 +1,9 @@
+use Mix.Config
+
+config :exchange,
+  message_bus_adapter: Exchange.Adapters.TestEventBus,
+  time_series_adapter: Exchange.Adapters.InMemoryTimeSeries,
+  environment: Mix.env()
```

Comment:
the :environment should be set for :dev, :test, :prod, etc, so i think it should be in `config.exs`